src/app/events/event-registration-option/event-registration-option.component.html

```
@@ -34,27 +34,33 @@ <h3 class="title-medium">{{ registrationOption().title }}</h3>
+          </p>
         } @else {
-          <button
-            (click)="register(registrationOption())"
-            mat-flat-button
-            [disabled]="registrationMutation.isPending()"
-          >
-            Register
-          </button>
+          @if (registrationOption().isPaid) {
+            <button
+              (click)="register(registrationOption())"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)